### PR TITLE
AG-12847 - drag and drop

### PR DESCRIPTION
**Pull request by peterjrreynolds**

### CODE REVIEW

File: community-modules/core/src/headerRendering/cells/columnGroup/headerGroupCellCtrl.ts [hunk 1]
```diff
@@ -464,96 +380,18 @@ export class HeaderGroupCellCtrl extends AbstractHeaderCellCtrl<
...
-        });
-
-        const result = childSuppressesMoving || this.gos.get('suppressMovableColumns');
-
-        return result;
+        return (
+            this.gos.get('suppressMovableColumns') ||
+            this.column
+                .getLeafColumns()
+                .some((column) => column.getColDef().suppressMovable || column.getColDef().lockPosition)
```
Comment (StephenCooper) [inline community-modules/core/src/headerRendering/cells/columnGroup/headerGroupCellCtrl.ts:393]: Share column.getColDef

File: community-modules/core/src/rendering/rowRenderer.ts [hunk 2]
```diff
@@ -200,13 +200,11 @@ export class RowRenderer extends BeanStub implements NamedBean {
...
         );
 
-        if (_isGroupRowsSticky(this.gos)) {
-            if (_isClientSideRowModel(this.gos) || _isServerSideRowModel(this.gos)) {
-                this.stickyRowFeature = this.createManagedBean(
-                    new StickyRowFeature(this.createRowCon.bind(this), this.destroyRowCtrls.bind(this))
-                );
-            }
-        }
+        this.stickyRowFeature = this.stickyRowService?.createStickyRowFeature(
```
Comment (StephenCooper) [inline community-modules/core/src/rendering/rowRenderer.ts:203]: Do we not need the row model checking?

**Review by StephenCooper — APPROVED** [review]
LGTM

File: community-modules/core/src/headerRendering/cells/columnGroup/headerGroupCellCtrl.ts (hunk 1 above)
Comment (peterjrreynolds) [reply community-modules/core/src/headerRendering/cells/columnGroup/headerGroupCellCtrl.ts:393]: Will address in later change

File: community-modules/core/src/rendering/rowRenderer.ts (hunk 2 above)
Comment (peterjrreynolds) [reply community-modules/core/src/rendering/rowRenderer.ts:203]: That check runs inside the sticky row service method

**Pull request closed without merging**